### ISSUE TITLE
[BUG] Bucket creation not possible

### ISSUE THREAD

**Issue by DavidsBees** [cobudget/cobudget#750]
**Describe the bug**
When trying to create a bucket in this round (within a group), I am able to fill all the data yet no bucket is created: https://cobudget.com/greaterthan/gt-cobudget-2022

The bucket is neither available in unpublished nor published state once leaving the editor

**To Reproduce**

1. Go to 'https://cobudget.com/greaterthan/gt-cobudget-2022'
2. Click on 'New Bucket'
3. Enter some data
4. Click on "publish"
5. Go to 'https://cobudget.com/greaterthan/gt-cobudget-2022'
6. See error

**Expected behavior**
Bucket visible in round

**Comment by gustavlrsn** [comment]
When trying to reproduce everything seems to work for me. Is there some error that you could printscreen or is the new bucket simply not there?

**Comment by DavidsBees** [comment]
Yep - didnt prompt me an error - just that there was no buckets (also after refreshing - login etc).

Now they are all there and the issue does not appear again
